bundlewrap/cmdline/debug.py

```diff
@@ -28,6 +28,17 @@ def bw_debug(repo, args):
     if args['command']:
         exec(args['command'], env)
     else:
-        set_completer(Completer(env).complete)
-        parse_and_bind("tab: complete")
+        # read 'bw debug' history, if it exists
+        history_filename = join(repo.path, '.bw_debug_history')
+        if isfile(history_filename):
+            readline.read_history_file(history_filename)
+
+        # set up tab completion
+        readline.set_completer(Completer(env).complete)
+        readline.parse_and_bind("tab: complete")
+
+        # launch interactive debug session
         interact(banner=banner, local=env)
+
+        # save history
+        readline.write_history_file(history_filename)
```

Comment:
There's also https://docs.python.org/3/library/readline.html#readline.append_history_file - but i have not gotten this to work on my laptop, i always get `AttributeError: module 'readline' has no attribute 'append_history_file'. Did you mean: 'read_history_file'?` (python 3.12.2 on MacOS, `readline._READLINE_VERSION` is 1026, whatever that means)